pravega/pravega
# Issue 402: Bugfix removes variable static members

**Change log description**
Removes variable static members from classes everywhere in the project.

**Purpose of the change**
While working on: Fixes #402
I discovered we had some dubious code that was using static members that were variable, which Findbugs was flagging correctly as problematic. To get the PR in I whitelisted the code. This change goes back and changes all the code to work the way it should and findbugs is happy with. 
The previous code was dubious because it depended on initialization order of classes to determine the outcome of certain variables. 

**What the code does**
The code does exactly what it did before. There are no functional changes. The only difference is that I have gone through each instance where a static variable was being modified and refactored the code so that the intended value was placed into it. In some cases this meant changing them to be member variables. In the case of ZKStream this meant factoring out all of the previously static methods and members into a helper class that is created when the previous code was initializing. (This a copy-paste)

**How to verify it**
All tests pass.
Now if you create a static non-final variable, the pre-config check will fail.
